src/main/java/org/kohsuke/github/GHEvent.java

```diff
@@ -69,6 +69,7 @@
     TEAM,
     TEAM_ADD,
     WATCH,
+    WEBHOOK_JOB,
```

Comment:
It's not `WEBHOOK_JOB` but `WORKFLOW_JOB`.

Reply:
Oh! I'll make the change, thanks.

Reply:
Also, my guess is that you will need to change things here: https://github.com/hub4j/github-api/blob/main/src/test/java/org/kohsuke/github/EnumTest.java#L31 .

Always run the tests after a change, especially in a code base you don't know as it's always full of surprises :).

Reply:
Thank you for the help!